packages/api/src/user.js

```diff
@@ -198,7 +198,26 @@ export async function userTokensGet (request, env) {
         }
       }
     }
-  `, { data: { user: request.auth.user._id } })
+  `, { user: request.auth.user._id })
 
   return new JSONResponse(res.findAuthTokensByUser.data)
 }
+
+/**
+ * Delete a user auth token. This actually raises a tombstone rather than
+ * deleting it entirely.
+ *
+ * @param {AuthenticatedRequest} request
+ * @param {import('./env').Env} env
+ */
+export async function userTokensDelete (request, env) {
+  const res = await env.db.query(gql`
+    mutation DeleteAuthToken($user: ID!, $authToken: ID!) {
+      deleteAuthToken(user: $user, authToken: $authToken) {
+        _id
+      }
+    }
+  `, { user: request.auth.user._id, authToken: request.params.id })
+
+  return new JSONResponse(res.deleteAuthToken, { status: 410 })
```

Comment:
`410 Gone` is cool, but i had to look it up, and status > 400 feels like the user did something wrong, like i tried to delete a thing that is already deleted. I think should probably just be another 200 or 204.